designs/developer/add-45/Pipelines-update/Pipelines-update.md

```diff
@@ -0,0 +1,53 @@
+---
+parent: Developer
+version: 4.5
+---
+# Add page- Updates
+
+Updates to the Add page include switching the order of the resources and pipelines section so resources come before pipelines.
+One more subtle update to the Add page is that the ‘Show pipeline visualization’ will stay open even if the user selects a new builder image or resource during their session.
```

Comment:
Since this is not replacing the +Add page but replacing the Pipelines section, could we change the wording to be more specific about the section? I would say something like "Updates to the Pipelines section of the +Add flow include moving this section below the Resources section. Also, within the Pipeline section itself, the ‘Show pipeline visualization’ component will stay open even if the user selects a new builder image or resource during their session.

Reply:
Yup I can update the title and wording so it's more Pipelines section specific and less about the overall page since that didn't change too drastically.